brainio_collection/packaging.py

```diff
@@ -67,6 +71,48 @@ def create_image_csv(proto_stimulus_set, target_path):
     return sha1
 
 
+def check_naming_convention(name):
+    assert re.match(r"[a-z]+\.[A-Z][a-zA-Z0-9]+", name)
+
+
+def check_image_naming_convention(name):
+    assert re.match(r"[a-zA-Z0-9]+_?(?!0)\d+\.(?:jpg|jpeg|png|mp4)|(?!0)\d+\.(?:jpg|jpeg|png|mp4)", name)
+
+
+def check_image_format(image, identifier):
+    assert image.mode in ['RGBA', 'RGB', 'LA', 'L'], f"{identifier}: incorrect image mode {image.mode}"
+    image_shape = np.array(image).shape
+    assert 3 == len(image_shape), f"{identifier}: incorrect shape {len(image_shape)}"
+
+    channels = {'RGBA': 4, 'RGB': 3, 'LA': 2, 'L': 1}
+    assert channels[image.mode] == image_shape[2], f"{identifier}: incorrect channels {image_shape[2]}"
+
+
+def check_image_numbers(stimulus_set):
+    image_numbers = [int(image_file_path[image_file_path.rfind('_') + 1:image_file_path.rfind('.')])
+                     for image_file_path in list(stimulus_set.image_paths.values())]
+    image_numbers.sort()
+    for i in range(len(image_numbers) - 1):
+        assert image_numbers[i] == image_numbers[i + 1] - 1, "StimulusSet files not sequentially numbered"
+
+
+def check_stimulus_set(stimulus_set):
```

Comment:
can we rename this to `check_experiment_stimulus_set` or so to make it clear that this is a special stimulus_set? Ideally also add a `"""` comment explaining what this does and what it's used for